pkg/reconciler/buildrun/resources/remote_artifacts.go

```diff
@@ -0,0 +1,58 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package resources
+
+import (
+	"strings"
+
+	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/shipwright-io/build/pkg/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	v1 "k8s.io/api/core/v1"
+)
+
+// renderRemoteArtifactsDownloadScript returns a slice of commands, a shell script, based on informed
+// BuildSources slice. Scripting lines are bind together with "&&".
+func renderRemoteArtifactsDownloadScript(sources []buildv1alpha1.BuildSource) []string {
+	script := []string{}
+	for _, source := range sources {
+		if len(script) > 0 {
+			script = append(script, "&&")
```

Comment:
Would it be useful at all to use `sh -e` instead of injecting `&&`s?

Reply:
Indeed 👍🏼 Thanks, that's more appropriate it.